### DISCUSSION TITLE
Next.JS not displaying correct favicon.

### DISCUSSION THREAD

**Discussion by samchouse** [vercel/next.js#22033] [Help] [answered]
Tree:
```
├── package.json
├── public
│   ├── favicon.ico
│   └── vercel.svg
├── README.md
├── src
│   ├── components
│   │   └── DarkModeSwitch.tsx
│   ├── pages
│   │   ├── api
│   │   │   └── hello.tsx
│   │   ├── _app.tsx
│   │   └── index.tsx
│   └── styles
│       └── globals.css
├── tsconfig.json
└── yarn.lock
```

The favicon should be the Vercel logo instead it is something else from another project.

**Comment by jamesmosier** [comment, marked as answer]
Hi @Xenfo. Have you cleared your cache or tried opening your Next.js project in a private browsing window? Favicons tend to get cached, especially when developing locally and using the same `localhost:3000` url between projects.

**Reply by samchouse** [reply]
It works in the private tab but my public is still the wrong icon.
